v2/src/components/VCard.vue

```diff
@@ -231,7 +232,6 @@ const hasDetails = computed(() => {
   box-shadow: var(--shadow);
   -webkit-box-shadow: var(--shadow);
   border-radius: var(--border-radius);
-  overflow: hidden;
```

Comment:
we need this overflow to be added back... because the border radius will not work without it... but you can change it to `overflow: auto;`